__init__.py

```
@@ -18,6 +18,21 @@
 # Import main transparency background remover nodes
 from .nodes import NODE_CLASS_MAPPINGS as NODES_MAPPINGS, NODE_DISPLAY_NAME_MAPPINGS as NODES_DISPLAY_MAPPINGS
 
+# Import standalone RemoveBackgroundAndResizeNode
+try:
+    from .RemoveBackgroundAndResizeNode import (
+        NODE_CLASS_MAPPINGS as REMBG_MAPPINGS,
+        NODE_DISPLAY_NAME_MAPPINGS as REMBG_DISPLAY_MAPPINGS,
+    )
+    REMBG_AVAILABLE = True
+except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)